js/customizer.js

```diff
@@ -0,0 +1,51 @@
+/* globals wp, UIEvent, jQuery, CodeMirror, _simpleCustomCSSCustomizerExports */
+/* exported simpleCustomCSSCustomizer */
+var simpleCustomCSSCustomizer = ( function( api, $ ) {
+	'use strict';
+
+	var self = {
+		editor: {},
+		textarea: {},
+		control: '',
+		element: ''
+	};
+
+	// Exports must be used to fill the previewData.
+	if ( 'undefined' !== typeof _simpleCustomCSSCustomizerExports ) {
+		$.extend( self, _simpleCustomCSSCustomizerExports );
+	}
+
+	self.init = function() {
+		$( document ).ready( function() {
+			var control;
+			control = api.control.instance( self.control );
+
+			// When our Customizer section is expanded, load CodeMirror.
+			api.section( control.section() ).container.on( 'expanded', function() {
+				self.loadCustomizer();
+			});
```

Comment:
Interesting use of `expanded` event here. I guess I forgot this was triggered. I think you should do this:

```js
api.section( control.section(), function( section ) {
    // ... Now we're guaranteed that section exists.
} )
```